packages/opencensus-core/test/test-console-exporter.ts

```diff
@@ -60,11 +44,15 @@ describe('NoopExporter', () => {
     it('should do anything', () => {
       const exporter = new NoopExporter();
       const rootSpan = new RootSpan(tracer);
-      const queue: RootSpan[] = [];
-      queue.push(rootSpan);
+      const queue: RootSpan[] = [rootSpan];
 
-      exporter.publish(queue);
-      assert.ok(true);
+      exporter.publish(queue)
+          .then(() => {
+            assert.ok(true);
```

Comment:
I would like to avoid this line entirely. If there are no conditions to test, that strongly suggests that there is no point in exercising this code path at all.

Reply:
In fact there is no need. I changed to just return `exporter.publish(queue)` as you suggested just to make sure we don't get any errors.